users/views.py

```diff
@@ -0,0 +1,188 @@
+"""
+This module contains
+"""
+# pylint: disable= no-self-use, no-member
+from django.contrib.auth.hashers import check_password, make_password
+from django.contrib.auth.password_validation import validate_password
+from django.core.exceptions import ValidationError
+from django.core.validators import validate_email
+from django.db.utils import IntegrityError
+from django.http import Http404
+from django.shortcuts import get_object_or_404
+from rest_framework import serializers, status
+from rest_framework.authentication import TokenAuthentication
+from rest_framework.authtoken.models import Token
+from rest_framework.decorators import authentication_classes, permission_classes
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from .models import User
+
+
+class UserSerializer(serializers.ModelSerializer):
```

Comment:
The serializer classes should be in another file which currently is empty in this PR.

Reply:
You asked me to move it from `serializers.py` to here